packages/ui/src/routes/send/SendPage.tsx

```diff
@@ -73,7 +89,14 @@ const SendPage = () => {
     }
 
     // Handlers
-    const onSubmit = handleSubmit((data: AddressFormData) => {
+    const onSubmit = handleSubmit(async (data: AddressFormData) => {
+        if (addContact) {
+            await addressBookSet(
+                data.address,
+                `Account (...${data.address.slice(data.address.length - 4)})`,
```

Comment:
small thing but we already have some utils that solve this and can reuse here: [formatHashLastChars](https://github.com/block-wallet/extension/blob/1a8fece141910d25286654a615f398bd1bd6b8f3/packages/ui/src/util/formatAccount.ts#L26)